osmosis-1/osmosis-1.assetlist.json

```diff
@@ -13838,6 +13838,57 @@
       "keywords": [
         "osmosis-unlisted"
       ]
-    }
+    },
+    {
```

Comment:
Please don't add directly to this generated file

Reply:
Apologies, Should we only add to osmosis.zone_assets.json then?